packages/common/src/projects/HubProjects.ts

```diff
@@ -0,0 +1,248 @@
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
+import { IHubProject } from ".";
+// Note - we separate these imports so we can cleanly spy on things in tests
+import { createModel, getModel, getModelBySlug, updateModel } from "../models";
+import { getUniqueSlug } from "../items/slugs";
+import { IModel, isGuid, cloneObject } from "..";
+import { IUserItemOptions, removeItem } from "@esri/arcgis-rest-portal";
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { createSlug, IPropertyMap, setSlugKeyword } from "./utils";
+import { PropertyMapper } from "./utils";
+
+/**
+ * Default values of a IHubProject
+ */
+const DEFAULT_PROJECT: Partial<IHubProject> = {
+  title: "No title provided",
+  properties: {},
+  tags: [],
+  typeKeywords: ["IHubProject", "HubProject"],
+  slug: "",
+  // status: "inactive",
+};
+
+export const PROJECT_ITEM_TYPE = "Web Mapping Application";
```

Comment:
This will change when QA has this item type... it's on devext now

Reply:
should this be namespaced as HUB_PROJECT_ITEM_TYPE or even better can it be HubProjects.ITEM_TYPE (or similar)

Reply:
I'll do one or the other... once the item type is on QA we can drop this construct though